File: polindrom_sayı.py
#-*-coding: utf-8-*-

def pol(n):
	if n == 1:
		print str(n),
	else:
		pol(n - 1)
		print str(n) ,
		pol(n - 1)
sayi = input("sayi kaç olsun: ")
pol(sayi)

File: polindrom_sayı.c
#include <stdio.h>

void pol(int n)
{
	if(n == 1)
		printf(" 1 ");
	else {
		pol(n - 1);
		printf("%d",n);
		pol(n - 1);
	}
}
int main()
{
	int sayi;
	printf("sayi kac olsun: ");
	scanf("%d",&sayi);
	getchar();
	pol(sayi);
	getchar();
	return 0;
}
